android-sdk/src/main/java/com/checkout/android_sdk/PaymentForm.java

```diff
@@ -586,14 +587,14 @@ private synchronized FramesLogger getFramesLogger() {
      */
     private void sendPaymentFormPresentedEvent(FramesLogger framesLogger) {
         post(() -> {
-            logPaymentFormPresentedEvent(framesLogger);
+            logPaymentFormPresentedEvent(framesLogger, new CheckoutTheme(mContext));
```

Comment:
Should be any Context null-check be done here for the cases when module is used without UI?
As far as I can see CheckoutTheme works with non nullable context.